### PR TITLE
Standardize "alternative to" proprietary services formatting

### PR DESCRIPTION
**Pull request by nodiscc**
When a project presents itself as an alternative to proprietary/SaaS products, mention it at the end of the description as `(alternative to X, Y, Z)`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,20 +128,20 @@ _Related: [Database Management](#database-management), [Personal Dashboards](#pe
...
 - [GoatCounter](https://www.goatcounter.com) - Easy web statistics without tracking of personal data. ([Source Code](https://github.com/arp242/goatcounter)) `EUPL-1.2` `Go`
 - [Matomo](https://matomo.org/) - Google Analytics alternative that protects your data and your customers' privacy. ([Source Code](https://github.com/matomo-org/matomo)) `GPL-3.0` `PHP`
 - [Metabase](https://metabase.com/) - Easy, open-source way for everyone in your company to ask questions and learn from data. ([Source Code](https://github.com/metabase/metabase)) `AGPL-3.0` `Java/Docker`
-- [Mixpost](https://mixpost.app/) - Self-hosted social media management software. Easily create, schedule, publish, and manage social media content in one place. Alternative to Hootsuite, Buffer, and other social media tools. ([Source Code](https://github.com/inovector/MixpostApp)) `MIT` `PHP/Docker`
+- [Mixpost](https://mixpost.app/) - Self-hosted social media management software. Easily create, schedule, publish, and manage social media content in one plac (alternative to Hootsuite, Buffer, and other social media tools). ([Source Code](https://github.com/inovector/MixpostApp)) `MIT` `PHP/Docker`
```
Comment (KieranRobson) [inline README.md:136]:
```suggestion
- [Mixpost](https://mixpost.app/) - Self-hosted social media management software. Easily create, schedule, publish, and manage social media content in one place (alternative to Hootsuite, Buffer, and other social media tools). ([Source Code](https://github.com/inovector/MixpostApp)) `MIT` `PHP/Docker`
```

File: README.md [hunk 2]
```diff
@@ -884,7 +884,7 @@ A [news aggregator](https://en.wikipedia.org/wiki/News_aggregator), also termed
 - [reader](https://github.com/lemon24/reader) - A Python feed reader web app and library (so you can use it to build your own), with only standard library and pure-Python dependencies. `BSD-3-Clause` `Python`
 - [RSS-Bridge](https://github.com/RSS-Bridge/rss-bridge) - Generate RSS/ATOM feeds for websites which don't have one. `Unlicense` `PHP`
 - [RSS Fulltext Proxy](https://github.com/Kombustor/rss-fulltext-proxy) - Mirrors RSS feeds to return the full content of the items, extracted from the website. `MIT` `Nodejs`
-- [RSS Monster](https://github.com/pietheinstrengholt/rssmonster) - RSS Monster is an easy to use web-based RSS aggregator and reader compatible with the Fever API, created as an alternative for Google Reader. `MIT` `PHP`
+- [RSS Monster](https://github.com/pietheinstrengholt/rssmonster) - RSS Monster is an easy to use web-based RSS aggregator and reader compatible with the Fever API (alternative to Google Reader). `MIT` `PHP`
```
Comment (KieranRobson) [inline README.md:887]:
```suggestion
- [RSS Monster](https://github.com/pietheinstrengholt/rssmonster) - An easy to use web-based RSS aggregator and reader compatible with the Fever API (alternative to Google Reader). `MIT` `PHP`
```

File: README.md [hunk 3]
```diff
@@ -1232,12 +1232,12 @@ _See also: [List of streaming media systems - Wikipedia](https://en.wikipedia.or
 - [Bsimp](https://github.com/akrylysov/bsimp) - Minimalistic S3-backed audio library. `Apache-2.0` `Go`
 - [euterpe](https://listen-to-euterpe.eu) - Self-hosted music streaming server with RESTful API and Web interface. ([Demo](https://listen-to-euterpe.eu/demo), [Source Code](https://github.com/ironsmile/euterpe)) `GPL-3.0` `Go`
 - [Funkwhale](https://dev.funkwhale.audio/funkwhale) - Modern, web-based, convivial, multi-user and free music server. `BSD-3-Clause` `Python/Django`
-- [GNU FM](https://gnu.io/fm/) - Running music community websites, alternative to last.fm. ([Source Code](https://git.savannah.gnu.org/cgit/librefm.git/)) `AGPL-3.0` `PHP`
+- [GNU FM](https://gnu.io/fm/) - Running music community websites (alternative to last.fm). ([Source Code](https://git.savannah.gnu.org/cgit/librefm.git/)) `AGPL-3.0` `PHP`
```
Comment (KieranRobson) [inline README.md:1235]: > https://fedi.network/fm

Seems to be a defunct project. The site points users to https://github.com/foocorp/hacienda

File: README.md [hunk 4]
```diff
@@ -1333,8 +1333,8 @@ Software that does not fit in another section.
...
 - [DOMJudge](https://www.domjudge.org/) - A system for running a programming contest, like the ICPC regional and world championship programming contests. ([Demo](https://www.domjudge.org/demo), [Source Code](https://github.com/DOMjudge/domjudge)) `GPL-2.0/BSD-3-Clause/MIT` `PHP`
 - [F-Droid](https://f-droid.org) - Server tools for maintaining an F-Droid repository system. ([Source Code](https://gitlab.com/fdroid/fdroidserver)) `AGPL-3.0` `Python/Docker`
-- [Flagsmith](https://flagsmith.com) - Flagsmith provides a dashboard, API and SDKs for adding Feature Flags to your applications. Alternative to LaunchDarkly. ([Source Code](https://github.com/flagsmith/flagsmith)) `BSD-3-Clause` `Docker`
-- [GO Feature Flag](https://gofeatureflag.org) - Simple, complete, and lightweight feature flag solution. Alternative to LaunchDarkly. ([Source Code](https://github.com/thomaspoignant/go-feature-flag)) `MIT` `Go`
+- [Flagsmith](https://flagsmith.com) - Flagsmith provides a dashboard, API and SDKs for adding Feature Flags to your applications (alternative to LaunchDarkly). ([Source Code](https://github.com/flagsmith/flagsmith)) `BSD-3-Clause` `Docker`
```
Comment (KieranRobson) [inline README.md:1336]: > https://github.com/Flagsmith/flagsmith/blob/main/LICENSE.md

This may need to be moved to `non-free`

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Couple things I picked up on that could be looked at. Happy to do a PR for the non-free change if needed once this one gets merged to avoid conflicts

File: README.md (hunk 3 above)
Comment (nodiscc) [reply README.md:1235]: Yes I also noticed this. I opened https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/39 that removes it, but we will have to wait for the full switch to https://github.com/awesome-selfhosted/awesome-selfhosted-data first.

File: README.md (hunk 4 above)
Comment (nodiscc) [reply README.md:1336]: The main LICENSE.md is actually the BSD-3-Clause license, **but** contents of the `api/permissions` directory are under a non-free license, and using them requires agreeing to Flagsmith's TOS (https://github.com/Flagsmith/flagsmith/blob/main/api/permissions/LICENSE.md).

So I agree this should be moved to non-free.md, and the license tag changed to `BSD-3-Clause/⊘ Proprietary`. I will do it in a separate PR.

**Review by briinsanne — CHANGES_REQUESTED** [review]
Password

**Pull request merged**